services/dedup/mirrorBadger/mirrorBadger.go

```diff
@@ -0,0 +1,41 @@
+package mirrorBadger
+
+import (
+	"github.com/rudderlabs/rudder-go-kit/config"
+	"github.com/rudderlabs/rudder-go-kit/stats"
+	"github.com/rudderlabs/rudder-server/services/dedup/badger"
+	"github.com/rudderlabs/rudder-server/services/dedup/scylla"
+	"github.com/rudderlabs/rudder-server/services/dedup/types"
+)
+
+type MirrorBadger struct {
+	badger *badger.Dedup
+	scylla *scylla.ScyllaDB
+}
+
+func NewMirrorBadger(conf *config.Config, stats stats.Stats) (*MirrorBadger, error) {
+	badger := badger.NewBadgerDB(conf, stats, badger.DefaultPath())
+	scylla, err := scylla.New(conf, stats)
+	if err != nil {
+		return nil, err
+	}
+	return &MirrorBadger{
+		badger: badger,
+		scylla: scylla,
+	}, nil
+}
+
+func (mb *MirrorBadger) Close() {
+	mb.scylla.Close()
+	mb.badger.Close()
+}
+
+func (mb *MirrorBadger) Get(kv types.KeyValue) (bool, int64, error) {
+	_, _, _ = mb.scylla.Get(kv)
```

Comment:
What if we handle the error here? In worst case scylla will get populated and badger will not, anyways we are throwing a panic in processor in such cases so everything will be retried.